README.md

```diff
@@ -1 +1,44 @@
+# Mellisa Live Album Launch 
+This is part of Microverse capstone project 1 to demonstrate understanding of HTML5, CSS3, JavaScript 
+and related frameworks.
 
+## Built With
+- HTML5
+- CSS3
+- JavaScript
+- Flexbox
+- CSS Grid
+
```

Comment:
- [ ] [Optional] Please leave a link to the recorded video presented in the README file.

Reply:
Thank you so much @ShahierNashaat! I think it's important to place the link there. I'll do so.